GliaWidgets/Sources/AlertManager/Alert/AlertViewController+Confirmation.swift

```diff
@@ -1,19 +1,58 @@
 import UIKit
 
 extension AlertViewController {
-    func makeConfirmationAlertView(
+    func makeLeaveConversationAlertView(
         with conf: ConfirmationAlertConfiguration,
         accessibilityIdentifier: String,
         confirmed: @escaping () -> Void
     ) -> AlertView {
-        let alertView = viewFactory.makeAlertView()
-        alertView.title = conf.title
-        alertView.message = conf.message
-        alertView.showsPoweredBy = conf.showsPoweredBy
-        alertView.accessibilityIdentifier = accessibilityIdentifier
-
+        let alertView = makeAlertView(
+            with: conf,
+            accessibilityIdentifier: accessibilityIdentifier,
+            confirmed: confirmed
+        )
         let alertStyle = viewFactory.theme.alert
+        var negativeButtonStyle = alertStyle.negativeNeutralAction
+        var positiveButtonStyle = alertStyle.positiveAction
+
+        if conf.switchButtonBackgroundColors {
+            negativeButtonStyle = alertStyle.positiveAction
+            positiveButtonStyle = alertStyle.negativeNeutralAction
+        }
+
+        negativeButtonStyle.title = conf.negativeTitle ?? ""
```

Comment:
Is it possible to fallback to some hardcoded or localized string?

Reply:
Fixed b4671b52d94685fd669a49387b9ac7495b674ad8